pages/workspace/tracking-plan/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: 'guide'
+title: 'Tracking Plan'
+# abstract: 'Learn about the tracking plan'
+---
+
+import Link from '../../../components/Link';
+
+A tracking plan is a document that defines the key stages of your customer life cycle and codifies a single source of truth for the data that supports it. Through this plan, you standardize your data management and capture better and cleaner data.
+
+When done well, this standardized plan will help you keep your analytics well-structured and your stakeholders informed for years to come.
+
+At a high-level, your plan should do three main things:
+
+- Clarify the events and properties that are relevant to your goals and customer journey
+- Explain the location in the codebase that your analytics should be added to make your developers’ lives easier
+- Summarize the reason for tracking each event and property and how they’ll be used to measure your progress.
+
+In an ideal world, these three things—and your tracking plan in totality—should be stored as a central, shareable document that your team can update together.
+
+Here is our <a href="https://www.avo.app/blog/avos-ultimate-tracking-plan-template-w-downloadable-worksheet">Ultimate Tracking Plan Template (w/ Downloadable Worksheet)</a>.
```

Comment:
Should we add a disclaimer here that this is a guide for the spreadsheet way of things (not how to use the Avo tracking plan), or is that clear enough in the blog post?

Reply:
Sounds good.
Maybe this does not belong here? Or how should we frame it?

Reply:
Good question, I say drop or frame it something like: "Not using Avo to manage your Tracking Plan? Check out our Ultimate Tracking Plan Template to learn how a tracking plan can be managed in a spreadsheet."